lib/v2/screens/app/app.dart

```diff
@@ -148,3 +152,65 @@ extension NavigatorStateExtension on NavigatorState {
     return isCurrent;
   }
 }
+
+Future<void> showAccountUnderRecoveryDialog(BuildContext buildContext) async {
```

Comment:
This dialog is not using our current dialogs, 
We have no designs for these dialogs. 

Do not review functionality here

Reply:
I suppose it will be as a component later.

BTW It would be good if we could do something similar as with the snackinfo to avoid the biolerplate of the showDialog method.

Reply:
this code has been removed

Reply:
and yes, ill add a helper to show dialogs and avoid boilerplate code